recipes/caerus/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.1.8" %}
+
+package:
+  name: caerus
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/caerus/caerus-{{ version }}.tar.gz
+  sha256: 5a23ea41a3a93865ae9f831bb50a642cfaad19ad8678f927cd630689573de31b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - setuptools
```

Comment:
Is setuptools actually needed at run time?

Reply:
probably not. threw it in hoping to fix the windows build but that for finding the issue about the README